app/components/works/work_form_component.html.erb

```diff
@@ -1,8 +1,9 @@
 <div id="editor">
   <h1 class="row">Deposit your work</h1>
-
-  <%= form_with model: work_form, url: [work_form.model.collection, work_form], data: { controller: 'progress' },
-      html: { class: 'needs-validation row', novalidate: true, multipart: true } do |f| %>
+  <%= form_with model: work_form, url: [work_form.model.collection, work_form],
+      local: true,
```

Comment:
Can you say more about converting this to a local form?

Reply:
The default is to do a remote form, however, our controller was not set up to respond with XHR requests.  By default in rails if you generate scaffold, it makes a local form like this.

Reply:
And it's your preference to convert this to a local form instead of tweaking the controller to respond to XHR? I'm wondering if folks will be confused to have a mix of local and remote forms.

Reply:
Do we have a mix?  Currently we just have a form that does not support validations. So if you submit and there is a validation error, you get no feedback.

Reply:
@jcoyne the collecton form is a remote form. I guess I've enjoyed the remote forms in my work on EDTF validation. How about we stick with remote forms for now to avoid the page refresh? Are you open to that?

Reply:
Are you confusing the client side validation with a remote for?  Those are two different things.

Reply:
@jcoyne yes, I am.